src/Equinox.MessageDb/MessageDb.fs

```diff
@@ -215,17 +215,15 @@ module Read =
             do! loop ()
             let act = Activity.Current
             if act <> null then act.AddBatches(batchCount).AddLastVersion(version) |> ignore
-            return version, Array.ofSeq result }
+            return version, state }
 
-    let private logBatchRead streamName t events (batchSize: int64) version (log: ILogger) =
-        let bytes, count = resolvedEventBytes events, events.Length
-        let reqMetric: Log.Measurement = { stream = streamName; interval = t; bytes = bytes; count = count}
-        let batches = (events.Length - 1)/ (int batchSize) + 1
+    let private logBatchRead streamName t version (log: ILogger) =
```

Comment:
needs to be passed the event count from the batch read
I also think at least a proxy for the size should be included in a bytes field
This will stll be true if it goes all OT - these things can be very useful for troubleshooting and/or analysis